### PR TITLE
Create FAQ.md

### PR DESCRIPTION
**Pull request by olayhemy**
A Frequently Asked Question section (FAQs) is without a doubt the most commonly used tool in self-service. Service users tend to feel that their question is the most important question, and they are right. With Frequently Asked Questions, service users will find answers to possible questions.

### CODE REVIEW

**Comment by olayhemy** [comment]
Please, merge this. I've made the necessary corrections

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+**Markor - Frequently Asked Questions**
+
+**Markor - FAQ about this project**
```
Comment (gsantner) [inline FAQ.md]: about this document

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+This Frequently Asked Question will be continually updated to accommodate new information with respect to new updates and changes on the project and will be used inside the App to help users find answers to their questions.
+
+
+**What is Markor?**
+
+Markor is an Android application developed by Gregor Santner. Markdown and todo.txt are two (and the currently only) special formats having syntax highlighting and a whole suite of text modification actions.
+
+
+**What size is Markor and hope it isn't harmful to device?**
```
Comment (gsantner) [inline FAQ.md]: this doesnt sound any good. remove at least the latter

File: FAQ.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Markor is a simple and lightweigh app that doesn't comsume much of your device memory or RAM when installed.
+ 
+
+**Where can I download Markor?**
+
+Markor can be downloaded from app store like Google Play Store and on F-droid app store.
```
Comment (gsantner) [inline FAQ.md]: see previous PR comment

File: FAQ.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+ 
+
+**Where can I download Markor?**
+
+Markor can be downloaded from app store like Google Play Store and on F-droid app store. 
+[Google Play Store](https://play.google.com/store/apps/details?id=net.gsantner.markor)
+[F-droid](https://f-droid.org/repo/net.gsantner.markor_20.apk)
+
+
+**Does Markor overwrites when installing?**
```
Comment (gsantner) [inline FAQ.md]: Typo/spelling

File: FAQ.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+No, nothing gets overwritten or deleted by markor unless told todo so (manually). Reinstalling/Updating does not change any document.
+
+
+**Does Markor request or share my personal information?**
+
+No personal data, like calendar or contacts information are requested or shared.
+
+
+**Deos Markor requires internet connection to run?**
```
Comment (gsantner) [inline FAQ.md]: typo

File: FAQ.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+The app doesn't use your internet connection unless external resources (e.g. pictures) are referenced in the user created markdown (text) document files. Otherwise it's working completly offline.
+
+
+**Can I backup my notes from Markor?**
```
Comment (gsantner) [inline FAQ.md]: notes -> documents

**Comment by olayhemy** [comment]
Hi, 
I've made the necessary corrections. Please, check and review.

**Comment by olayhemy** [comment]
@gsantner
Kindly merge the FAQ

**Pull request closed without merging**